src/NUnit3TestAdapterInstallV2/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="NUnit.NUnit3TestAdapter" Version="4.0.0.0" Language="en-US" Publisher="Terje Sandstrom" />
+    <Identity Id="NUnit.NUnit3TestAdapter" Version="3.6.0.1" Language="en-US" Publisher="Terje Sandstrom" />
```

Comment:
We have a general approach to doing releases that involves not mixing the actual code fixes with the release changes. Permanent code changes are done on the master branch. Release changes, that should not go to master, on the release branch. This is all pretty well documented, with perhaps a few things that need updating.

The master branch needs to stay at 4.0, which we decided would be our next major/minor release.

The last release we did was 3.5. So making this new release 3.6.1 is going to be a bit confusing to users. I think this number choice is a side effect of that fact that you are making the version change on master rather than the release branch. We should have decided in advance whether this was to be release 3.5.1 or release 3.6. I guess there are arguments either way - it depends whether you consider "Works on dev 15" to be a bug fix or a feature.